File: Shared/Services/UC/CommonUCService.cs
using Shared.Requests;
using Shared.Responses;

namespace Shared.Services.UC
{
public class CommonUCService : BaseUCService<CommonRequest, CommonResponse>
{
    protected override string GetControllerName() => "Common";

    public async Task<CommonResponse> FromToAll() => await Request<CommonResponse>(EHTTPRequest.Get, nameof(FromToAll));
}
}

File: Web/src/services/uc/CommonUCService.ts
import { CommonRequest } from "../../models/requests/CommonRequest";
import { CommonResponse } from "../../models/responses/CommonResponse";
import { BaseUCService, EHTTPRequest } from "./BaseUCService";

export class CommonUCService extends BaseUCService<
  CommonRequest,
  CommonResponse
> {
  public constructor() {
    super();
  }

  protected override GetControllerName() {
    return "Common";
  }

  public async FromToAll() {
    return await super.Request(EHTTPRequest.Get, "FromToAll");
  }
}
